frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -969,6 +968,18 @@ export const ClusterSettingsAlerts: React.FC<ClusterSettingsAlertsProps> = ({
   const pausedMCPs = machineConfigPools
     .filter((mcp) => !isMCPMaster(mcp))
     .filter((mcp) => isMCPPaused(mcp));
+  if (!canUpgrade) {
+    return (
+      <Alert
+        variant="info"
+        isInline
+        title={t(
+          'public~Control plane is externally managed.', // TODO: final copy
```

Comment:
@megan-hall, final copy is needed for the alert.

Reply:
@rhamilto minimally we should change the alert to read: "Control-plane is hosted." Working with docs now to see if there will be an external link we want to surface here or if we need any further explanation. I will circle back early next week.